extension/chrome/settings/modules/my_key.ts

```diff
@@ -68,13 +79,35 @@ View.run(class MyKeyView extends View {
     }
   }
 
+  private downloadRevocationCert = async (enteredPP?: string) => {
```

Comment:
I'm thinking that the code could really simplify we used `elements/passphrase.htm` module instead.

Or maybe if we could make Sweetalert prompt where user enters some info, then we could use that for pass phrases which is much simpler.

But for now we can go with this.